doc/dftb+/manual/dftbp.tex

```diff
@@ -2756,6 +2757,9 @@ \subsubsection{DftD3 optional settings}
 \item[\is{Threebody}] Whether three-body contributions should be included in the
   dispersion interactions.
 
+\item[\is{AtomicNumbers}]
+  Allows overwrite of atomic numbers associated with a species name.
```

Comment:
Probably needs a syntax use example as a verbatim block

Reply:
Added an example, but I think it is more an expert feature one learns from the provided tests in the testsuite.